packages/panzer/adapters-stk/src/Panzer_STK_LocalMeshUtilities.hpp

```diff
@@ -46,7 +46,7 @@
 namespace panzer
 {
 template <typename LO, typename GO>
-class LocalMeshInfo;
+struct LocalMeshInfo;
```

Comment:
Is the actually type a 'struct' and not a 'class'?

Reply:
yes - clang picked up on the fact that the object was a "struct" but some files had forward declarations using "class". I just cleaned up all the forward declares.